src/moscot/problems/base/_compound_problem.py

```diff
@@ -371,7 +371,7 @@ def push(self, *args: Any, **kwargs: Any) -> ApplyOutput_t[K]:
         %(scale_by_marginals)s
 
         kwargs
-            keyword arguments for :meth:`moscot.problems.CompoundProblem._apply`.
+            keyword arguments for :meth:`~moscot.problems.CompoundProblem._apply`.
```

Comment:
If you really wanna refer to the docs, `apply` should be either made public, or you should explicitly tell sphinx to document this private method (currently links to nothing). Am OK with both options, the former is easier.

Reply:
I did an intermediate solution now , added TODO, and will think about the documentation of private functions later